rohmu/compressor.py

```diff
@@ -23,10 +24,10 @@
     zstd = None  # type: ignore
 
 
-def CompressionFile(dst_fp, algorithm, level=0, threads=0):
+def CompressionFile(dst_fp: FileLike, algorithm: str, level: int = 0, threads: int = 0) -> FileLike:
```

Comment:
The level values depends on the compression algorithm. Most have a parameter from 0 to 9, ZSTD has a range 1-22. But, IMHO, I don't think that using `Literal[int]` here would give us anything. It would only make harder to write code that uses this function without much benefits. 

The `Literal[str]` in the `atomic_opener` makes much more sense because the `@overload` can be used to specify `TextIO` vs `BinaryIO` in return type, but whether the compression level is `1`, `100` or 1 million doesn't really change much and seems only a configuration parameter, so I'd rather leave this as plain `int`.